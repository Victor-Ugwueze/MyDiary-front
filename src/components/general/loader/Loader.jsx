// react libraries
import React from 'react';

// third-pary libraries
import LoadingIndicator from 'react-loader-spinner';

const Loader = () => (
  <div className="loader-spiner">
    <LoadingIndicator type='Oval' color='#00BFFF' height='100' width='100' />;
  </div>
);

export default Loader;
