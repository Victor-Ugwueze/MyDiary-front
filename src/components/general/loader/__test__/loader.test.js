// React Libraries
import React from 'react';

// third-party libraries
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


// components
import Loader from '../Loader';


configure({
  adapter: new Adapter(),
});

describe('Renders the loader-spinner', () => {
  it('should render the list section of dashboard page', () => {
    const component = shallow(
      <Loader />
    );
    expect(component).toMatchSnapshot();
  });
});
