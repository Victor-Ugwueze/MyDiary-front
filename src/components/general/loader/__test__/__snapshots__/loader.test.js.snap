// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the loader-spinner should render the list section of dashboard page 1`] = `
<div
  className="loader-spiner"
>
  <Loader
    color="#00BFFF"
    height="100"
    type="Oval"
    width="100"
  />
  ;
</div>
`;
